test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -225,4 +225,38 @@ Describe "Get-Help should find help info within help files" -Tags @('CI', 'Requi
             Remove-Item $helpFilePath -Force -ErrorAction SilentlyContinue
         }
     }
+
+    # There is a bug specified to Tranvis CI that hangs the test if "get-help" is used to search pattern string. This doesn't repro locally. 
+    # This occurs even if Unix system just returns "Directory.GetFiles(path, pattern);" as the windows' code does.
+    # Since there's currently no way to get the vm from Tranvis CI and the test PASSES locally on both Ubuntu and MacOS, excluding pattern test under Unix system.
+    It "Get-Help should find pattern help files" -skip: (-not $IsWindows){
+        $helpFile1 = "about_testCase1.help.txt"
+        $helpFile2 = "about_testCase.2.help.txt"
+        $culture = (Get-Culture).Name
+        $helpFolderPath = Join-Path $PSHOME $culture
+        $helpFilePath1 = Join-Path $helpFolderPath $helpFile1
+        $helpFilePath2 = Join-Path $helpFolderPath $helpFile2
+
+        if (!(Test-Path $helpFolderPath))
+        {
+            $null = New-Item -ItemType Directory -Path $helpFolderPath -ErrorAction SilentlyContinue
+        }
+
+        try
+        {
+            # Create at least one help file matches "about*" pattern
+            $null = New-Item -ItemType File -Path $helpFilePath1 -Value "about_test1" -ErrorAction SilentlyContinue
+            $null = New-Item -ItemType File -Path $helpFilePath2 -Value "about_test2" -ErrorAction SilentlyContinue           
+            Get-Help about_testCas?1 | Should Match "about_test1"
+            Get-Help about_testCase.? | Should Match "about_test2"
+            (Get-Help about_testCase*).Count | Should Be 2
+            Get-Help about_testCas?.2* | Should Match "about_test2"
```

Comment:
Agree with @adityapatwardhan, this is a good example to use the `-TestCases` parameter of Pester.  Look at other test code in this repo as a sample.